src/libraries/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -802,7 +803,7 @@ public static IEnumerable<object[]> UseSocketsHttpHandler_WithIdFormat_MemberDat
             yield return new object[] { false, ActivityIdFormat.W3C };
         }
 
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
+        /*[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
```

Comment:
If memory serves, I removed these since all the combinations of `SendAsync_ActivityIsCreatedIfRequested` cover these as well.

Reply:
I forgot to delete them, oops.